asset/css/controls.less

```diff
@@ -30,3 +30,54 @@
   }
 }
 
+
+// Style
+
+.control-button {
+  .appearance("none");
+  background: none;
+  border: none;
+  color: @icinga-blue;
+  .rounded-corners();
+
+
+  &:hover, &:focus, &.active {
+    background-color: @gray-lighter;
+  }
+
+  i.icon:before {
+    color: inherit;
+  }
+}
+
+// Layout
+
+.control-button {
+  padding: 0 .5em;
+
+  > i.icon {
+    display: inline-flex;
+    align-items: center;
+    height: 100%;
+  }
+
+  i.icon:before {
+    margin-right: 0;
+  }
+}
+
+span.control-button {
+  color: @gray-light;
+
+  &:hover {
+    background: none;
+  }
+}
+
+.controls .icinga-form.inline.sort-control .control-group:last-child {
+  margin-right: 0;
+}
+
+.controls .icinga-form.inline.sort-control .control-group .control-button:last-child {
+  margin-right: -.5em;
+}
```

Comment:
Two things:

* It's now perfectly obvious that these exceptions have no place here. The `.controls` widget is an Icinga DB Web thing. `.icinga-form.inline .control-group` an Icinga Web 2 (module) thing. It's not a layout rule originating from the `.sort-control` either, it's necessary because it's part of a column's controls and happens to be last visible element in a row. All of this means, this CSS must not be part of ipl-web, but of Icinga DB Web.
* You can reduce both sections to one by applying the `margin-right: 0;` to the `.control-button`. There's already a negative margin on the `.control-group` which does the exact same job as yours.

Reply:
✅